lib/exercism/user_statistics.rb

```diff
@@ -0,0 +1,30 @@
+class UserStatistics
+  def self.of(username)
+    by_username(username)
+  rescue Exception => e
+    Bugsnag.notify(e, {username: username})
+    id
```

Comment:
If we return the `id` here we will sometimes get a hash with user/statistics, and sometimes get an integer. We should probably return the same type of thing, always.